### PR TITLE
feat:Added homepage

### PR DESCRIPTION
**Pull request by Hardiegogo**
Added homepage:
image
image

### CODE REVIEW

**Review by AmishMishra11 — APPROVED** [review]
Clean code, good class names, you can merge

File: index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                </nav>
+            </div>
+        </header>
+        <main>
+            <div class="hero-section p-24">
+                <div class="hero-left">
+                    <h2>
+                        Want to answer <br> some super interesting Quizes? <br>
+                        Welcome to <br> <span class="primary-text">Modi</span>-Quiz. <br>
+                        <button class="btn btn-primary btn-quiz-hero">Explore quizes</button>
```
Comment (shivanigangadharan) [inline index.html]: I think this button can be linked to the component below (categories of quizzes)

**Review by shivanigangadharan — COMMENTED** [review]
Apart from 1 thing mentioned, rest of the page looks good. Code can be merged after linking the button.

File: index.html (hunk 1 above)
Comment (Hardiegogo) [reply index.html]: Got it, thanks Shivani.

**Pull request merged**